core/tests/mod.rs

```diff
@@ -0,0 +1,778 @@
+use std::{collections::{HashSet, HashMap}, sync::{Arc, Mutex,},};
+use std::str::FromStr;
+use std::path::Path;
+use std::time::Instant;
+
+use iroha_config::{
+    base::proxy::{LoadFromEnv, Override},
+    iroha::{Configuration, ConfigurationProxy},
+};
+use iroha_core::{kura::Kura, prelude::*, queue::Queue, sumeragi::*, sumeragi::message::*, sumeragi::view_change::*};
+
+use iroha_data_model::prelude::*;
+use iroha_data_model::block::*;
+use iroha_genesis::{GenesisNetwork, RawGenesisBlock, GenesisNetworkTrait};
+use iroha_core::smartcontracts::Registrable;
+use rand::seq::IteratorRandom;
+use rand::SeedableRng;
+
+use iroha_core::sumeragi::main_loop::Sumeragi;
+
+use iroha_data_model::transaction::Accept;
+
+
+/// Get a standardised key-pair from the hard-coded literals.
+///
+/// # Panics
+/// Programmer error. Given keys must be in proper format.
+pub fn get_key_pair() -> KeyPair {
```

Comment:
This comment makes me think that this function accepts some kind of keys but it's not. I'd replace it like this:

```suggestion
/// Programmer error. Hardcoded keys must be in proper format.
pub fn get_key_pair() -> KeyPair {
```